Core/include/Acts/Seeding/SeedFinderConfig.hpp

```diff
@@ -188,12 +188,19 @@ struct SeedFinderConfig {
   Delegate<Acts::Vector3(const SpacePoint&)> getStripCenterDistance;
   /// Returns position of the center of the top strip.
   Delegate<Acts::Vector3(const SpacePoint&)> getTopStripCenterPosition;
+  /// Delegate for specifying experiment specific cuts on space points
+  /// before filling the grid. Currently accepts three parameters
+  Delegate<bool(const SpacePoint&)> spacePointSelector{
+      DelegateFuncTag<voidSpacePointSelector>{}};
+
+  static bool voidSpacePointSelector(const SpacePoint&) { return true; }
```

Comment:
I am surprised that the lookup works. Alternatively we could also bind a lambda in this case to avoid having a static function in the config

Reply:
True, this is actually somewhat surprising.

Reply:
Shall I change to lambda function or keep it like this?

Reply:
only a small preference from my side. what do you think @paulgessinger ?

Reply:
final suggestion here @paulgessinger ?